### ISSUE TITLE
model of segmentaion

### ISSUE THREAD

**Issue by lyc6749** [JialianW/TraDeS#18]

**Comment by lyc6749** [comment]
I use coco to pretrain a model with seg and track, but the loss of cost_volume is `nan.```

**Comment by hulihan-start** [comment]
I also want to use this model for segmentation and segmentation... Do you know how to set the options?

**Comment by qianmin** [comment]
> I use coco to pretrain a model with seg and track, but the loss of cost_volume is `nan.```

Well, you said you have trained this seg model. So do you know how to use the seg model author provided to seg our own video? looking forward to your reply.
